fractions/microprofile/microprofile-jwt/pom.xml

```diff
@@ -25,6 +25,7 @@
     <swarm.fraction.stability>stable</swarm.fraction.stability>
     <swarm.fraction.tags>Eclipse MicroProfile,JWT,Security,Web</swarm.fraction.tags>
     <version.jose4j>0.6.0</version.jose4j>
+    <version.smallrye.jwt>1.1.0</version.smallrye.jwt>
```

Comment:
This should go into the `build-parent` pom: https://github.com/thorntail/thorntail/blob/master/build-parent/pom.xml#L114-L116

Reply:
Done

Reply:
Not really. You've only copied the version property to build-parent. The property here should be removed. Also `io.smallrye:smallrye-jwt` dependency should be added to `<dependencyManagement>` section in build-parent; there is a `<!-- SmallRye -->` comment where other MPs reside.